src/test/java/io/ebean/UpdateQueryTest.java

```diff
@@ -30,6 +33,7 @@ public void basic() {
   }
 
   @Test
+  @IgnorePlatform(Platform.SQLSERVER)
```

Comment:
TableAlias does not work on sqlserver, only when the (slower) select mode=cursor is used. (but then other tests will fail)